### PR TITLE
Bug/ch1687/need to scroll right to see the x on cookie v2

### PR DESCRIPTION
**Pull request by SamMed1**

### CODE REVIEW

**Comment by joesb** [comment]
I'll merge #124, which reverts #123 that I merged coz through my limited understanding. That may cause a conflict in the `styles.css` file here that'll, then need to be rebuilt I guess.

I'll leave @lewisnyman to review this, as my knowledge is lacking.

**Comment by joesb** [comment]
Aah, #124 has a merge conflict in `styles.css` anyway, so maybe this fix and updated CSS file makes #124 redundant?

File: web/themes/custom/bevan/sass/components/_layout.scss [hunk 1]
```diff
@@ -11,8 +11,9 @@
 }
 
 @media only screen and (max-width : 768px) {
-	.contextual-region {
+	.govuk-table {
```
Comment (lewisnyman) [inline web/themes/custom/bevan/sass/components/_layout.scss]: I had a quick look to see if changing a table to display block doesn't anything weird and aparently [it does!](https://developer.paciellogroup.com/blog/2018/03/short-note-on-what-css-display-properties-do-to-table-semantics/)

I think we could try using the .views-element-container wrapper class for this instead?

**Comment by SamMed1** [comment]
Changed css class styles applied to  @lewisnyman

**Comment by lewisnyman** [comment]
👍 🚢

**Pull request merged**